phac-nml/irida-next
# Namespace group link UI destroy

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR add in the UI functionality to delete a namespace group link for both groups and projects.

Merge after #178 

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/25466211/0004f8c5-bc8e-414f-940d-fee04f2c8f85)

![image](https://github.com/phac-nml/irida-next/assets/25466211/af543974-7194-4d50-b7a3-b38b326194ef)

![image](https://github.com/phac-nml/irida-next/assets/25466211/7c199f0a-6076-4211-8df1-fa712f4f1fa1)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Login as any user that groups and projects underneath these groups
2. Go to a group's members page and click on the `Groups` tab
3. Click on the `ellipsis` and then click `Unlink` for a direct member namespace group link. You should only be able to unlink direct members
4. Click `Confirm` in the confirmation window
5. The namespace group link should be deleted and you should see a flash message indicating successful deletion
6. Repeat with a project

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
